### PR TITLE
fix(@nestjs/microservices): gRPC stream errors fix

### PR DESCRIPTION
**Pull request by AlexDaSoul**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ *] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [* ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[* ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
gRPC stream returning errors

## Does this PR introduce a breaking change?
```
[ ] Yes
[* ] No
```

### CODE REVIEW

**Comment by anton-alation** [comment]
Maybe the test for such behavior should be added?

**Comment by AlexDaSoul** [comment]
> Maybe the test for such behavior should be added?
Ready

File: packages/microservices/test/server/server-grpc.spec.ts [hunk 1]
```diff
@@ -274,20 +298,20 @@ describe('ServerGrpc', () => {
...
+        },
       };
       const services = server.getServiceNames(grpcPkg);
       expect(services.length).to
         .be.equal(
         2,
-        'Amount of services collected from namespace should be equal 2'
+        'Amount of services collected from namespace should be equal 2',
       );
       expect(services[0].name).to.be.equal('A');
```
Comment (LimarenkoDenis) [inline packages/microservices/test/server/server-grpc.spec.ts]: test

**Comment by kamilmysliwiec** [comment]
I'll review soon. Thanks!

**Comment by kamilmysliwiec** [comment]
It has been fixed already

**Pull request closed without merging**